stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/view/MediaAttachmentView.kt

```diff
@@ -44,7 +44,11 @@ internal class MediaAttachmentView : ConstraintLayout {
         defStyleRes
     )
 
-    fun showAttachment(attachment: Attachment, andMoreCount: Int = NO_MORE_COUNT) {
+    fun showAttachment(
+        attachment: Attachment,
+        andMoreCount: Int = NO_MORE_COUNT,
+        shouldSetClickListeners: Boolean = true,
```

Comment:
Not the best solution - we shouldn't set those listeners if not needed but I didn't want to refactor this in the scope of this PR

Reply:
Is there a strong reason for using `MediaAttachmentView` inside `LinkAttachmentView`, by the way? Maybe we could replace it with a simpler `ImageView`.

Reply:
I think it could be replaced easily. Let met check that